coderedcms/models/page_models.py

```diff
@@ -545,6 +547,18 @@ def body_preview(self):
         return body[:200] + "..."
 
 
+    @property
+    def page_ptr(self):
+        """
+        Overwrite of `page_ptr` to make it compatible with wagtailimportexport.
+        """
+        return self.base_page_ptr
+
+    @page_ptr.setter
+    def page_ptr(self, value):
+        self.base_page_ptr = value    
+
```

Comment:
I'm assuming this override will not mysteriously break things in wagtail? Similar to the `get_children` that we previously had to quit overriding.

Reply:
It shouldn't.  If anything it'll help with compatibility with other wagtail packages.

Reply:
Oh, ok. I don't know much about the page_ptr. Just wanted to double check.